respy/state_space.py

```diff
@@ -279,6 +268,30 @@ def get_continuation_values(self, period=None, indices=None):
 
         return continuation_values
 
+    def _get_transition_probabilities(self, period=None, indices=None):
```

Comment:
Should we try to unify this with continuation values? It seems like following a similar logic!

Reply:
The control flows are a little bit dissimilar and, thus, I did not want to merge them. it is better to separate the logic.